app/controllers/email_alert_subscriptions_controller.rb

```diff
@@ -7,18 +7,36 @@ class EmailAlertSubscriptionsController < ApplicationController
 
   def new
     if content
-      @signup = SignupPresenter.new(content)
+      @signup = signup_presenter
     else
       error_not_found
     end
   end
 
   def create
-    redirect_to email_alert_signup_api.signup_url
+    if valid_choices?
+      redirect_to email_alert_signup_api.signup_url
+    else
+      @signup = signup_presenter
+      @error_message = "Please choose an email alert"
+      render action: :new
+    end
   end
 
 private
 
+  def valid_choices?
+    if available_choices.present? && chosen_options.blank?
```

Comment:
Would this method body be better as `available_choices.blank? || chosen_options.present?`